patch/increase-maximum-gpio-number-for-x86-64.patch

```diff
@@ -0,0 +1,36 @@
+Backport from Linux 5.15
+
+commit d7109fe3a0991a0f7b4ac099b78c908e3b619787
+Author: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
+Date:   Thu Aug 26 18:03:17 2021 +0300
+
+    x86/platform: Increase maximum GPIO number for X86_64
```

Comment:
Hi Sai, May I know the purpose of changing file name?

Reply:
It matches the git commit message summary.